### ISSUE TITLE
Ajax Alarm - trigger issue

### ISSUE THREAD

**Issue by alucryd** [eavanvalkenburg/pysiaalarm#18]
Hi there, I'm trying to set up some automations when my ajax alarm is armed and disarmed. While the `disarmed` trigger works as intended, there is no `armed` or `armed away` trigger, only `arming`, which doesn't appear to do anything when I arm the alarm. Also the `arming` trigger doesn't show the device name while the 2 others do. Is this a known issue?

![image](https://user-images.githubusercontent.com/1561084/129407801-6736ee48-dd69-438d-8934-d7f653ba5140.png)

**Comment by eavanvalkenburg** [comment]
I'm not sure, there is nothing in the SIA specific to those device triggers, that all is generic alarm control panel stuff. Also, make sure you switch to the official integration, I'll deprecate this repo at some point!

**Comment by alucryd** [comment]
I am on the official integration, wasn't sure where to post the issue though. Should I report over at the hass repo instead?

**Comment by eavanvalkenburg** [comment]
yeah, if it is really HA specific and you are using the official integration, then report there, if this is still not working!
